### PR TITLE
Add RFC 0007 -- Accessibility and Inclusion.

### LINKED ISSUE #18: Create accessibility and inclusion RFC
Accessibility and inclusion are two cornerstones of our work at CDS. We should document what that means and provide pointers and guidance in the discussion board on the hows.

See https://digital.canada.ca/a11y/ for more info.

### PR DESCRIPTION
**Pull request by dj2**
# Summary | Résumé

This PR adds an RFC around the accessiblity and inclusion standards
the developer community works to uphold.

Fixes #18

### CODE REVIEW

**Comment by dj2** [comment]
This content is primarily copied from work by @JuliannaR so I can't take credit for anything but the mistakes putting it into the RFC format.

**Review by wmoussa-cds — APPROVED** [review]
lgtm

File: rfcs/0007-accessibility-and-inclusion.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+it better than it was yesterday.
+
+
+## Motivation
+We build services to for anyone that wants to interact with the Canadian
+Government which, by its nature, includes people who require the accessibility
+features of our sites. In order to meet our mission we MUST build accessibility
+and inclusivity into our software.
+
+Further, there the [Canadian Government Standard on Web Accessibility](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=23601) requires sites to be accessible and to meet certain
```
[the remark this answers is not in the document]
Comment (JuliannaR) [reply rfcs/0007-accessibility-and-inclusion.md:34]: with the Canadian government should probably say with the Government of Canada

**Review by JuliannaR — APPROVED** [review]
I think we can tweak this later, but its a great start.

File: rfcs/0007-accessibility-and-inclusion.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+development, design, and accessibility.
+
+Inclusivity is a high priority in all of our work. Ensuring that everyone can
+interact with our services in a way that meets their individual needs and
+promotes their independence and dignity is important to us. Our goal is to make
+it better than it was yesterday.
+
+
+## Motivation
+We build services to for anyone that wants to interact with the Canadian
```
[the remark this answers is not in the document]
Comment (dj2) [reply rfcs/0007-accessibility-and-inclusion.md]: done

File: rfcs/0007-accessibility-and-inclusion.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dj2) [reply rfcs/0007-accessibility-and-inclusion.md]: done

**Pull request merged**